packages/extensions/openapi-to-typespec/src/emiters/emit-main.ts

```diff
@@ -33,6 +33,7 @@ function getArmServiceInformation(program: TypespecProgram) {
     `import "@azure-tools/typespec-azure-core";`,
     `import "@azure-tools/typespec-azure-resource-manager";`,
     `import "./models.tsp";`,
+    `import "./client.tsp";`,
```

Comment:
```suggestion
```

do not import client.tsp in main.tsp. instead, import main.tsp in client.tsp and compile with client.tsp.

Reply:
and if there is nothing to generate for client.tsp, we will not generate it.

Reply:
removed. thanks